### PR TITLE
Basic tests for wallet connect expected state

### PR DESCRIPTION
**Pull request by asnaith**
related to #422

### CODE REVIEW

**Comment by asnaith** [comment]
I found these tests in a stash, think I started working on them before realizing we couldn't go much further with wc 

Thought we might as well add these basic checks as they are.

File: packages/ui/cypress/tests/wallet-connect.cy.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    settingsPage.connectDappButton().should('be.disabled')
+  })
+
+  it('can see expected wc state when wallet is connected', () => {
+    cy.setupAndVisit({
+      url: settingsPageUrl,
+      extensionConnectionAllowed: true,
+      injectExtensionWithAccounts: [testAccounts['Multisig Member Account 1']]
+    })
+    topMenuItems.settingsButton().click()
```
Comment (Tbaut) [inline packages/ui/cypress/tests/wallet-connect.cy.ts:29]: That one shouldn't be needed if you're connected already and visited the settings page.
```suggestion
```

**Review by Tbaut — APPROVED** [review]
Looks good to me, just one line that shouldn't be needed.

File: packages/ui/cypress/tests/wallet-connect.cy.ts (hunk 1 above)
Comment (asnaith) [reply packages/ui/cypress/tests/wallet-connect.cy.ts]: Ah yeah originally I was click to get through to the settings page but then changed that approach and forgot to take this out. Thanks

**Pull request merged**